pkg/encryption/decrypt_store.go

```diff
@@ -0,0 +1,96 @@
+// Copyright 2020 The Swarm Authors. All rights reserved.
```

Comment:
I agree that decryptingStore should be in its own package like `pkg/encrytpion/store`. And alos to depend on encrytion.Interface in order to be tested without testing the encryption, but just the store.